memote-report-app/src/index.html

```diff
@@ -10,6 +10,7 @@
   <script type="text/javascript" async
   src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML">
 </script>
+<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/taucharts@2/dist/taucharts.min.css">
```

Comment:
I'm a bit skeptical to requiring external resources in the report, but I realize that wasn't introduced in this PR though.